conda_build/cli/main_build.py

```diff
@@ -153,6 +153,11 @@ def parse_args(args):
         help=("Build root folder.  Equivalent to CONDA_BLD_PATH, but applies only "
               "to this call of conda-build.")
     )
+    p.add_argument(
+        "--no-verify",
+        action="store_false",
+        help=("do not run verification on recipes or packages when building")
```

Comment:
This is very unintuitive to me.  Since you have some implicit boolean logic inversion going on here, it's hard for me to figure out.

What this code says to me right now is:
- if you pass --no-verify, then it sets no_verify to false
- if you don't pass it, no_verify is true

What might be clearer would be:

```
 p.add_argument(
         "--no-verify",
         action="store_false",
         dest="verify",
         help=("do not run verification on recipes or packages when building")
```

and then change the reference below to args.verify.